### DISCUSSION TITLE
List of global color presets for the Color field

### DISCUSSION THREAD

**Discussion by scoch5** [craftcms/cms#14500] [Ideas] [answered]
A further improvement for Color field with colors presets would be to have the list of presets at site or craft installation level to avoid repeating the same presets for each field. Perhaps it could be a configuration to be inserted in the config/general.php so that every single field can use the default one or have one just for itself.
It would become very useful in case matrix blocks have background color or text color configurations and help editors maintain a consistent site palette.

**Comment by brandonkelly** [comment, marked as answer]
I don’t think this idea is necessary anymore, now that Craft 5 allows you to create multiple instances of the same field, and Matrix fields share the same entry types as sections. So you can define a Color field with your preferred presets once, and reuse it wherever you want.
